scripts/run-figures.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# 
+# Run all figure making scripts. 
+
+# Find current directory based on this script
+cd "$(dirname "${BASH_SOURCE[0]}")" 
+ 
+# Make an output folder for all directories
+mkdir figures
```

Comment:
This would put `figures` in the scripts folder and also fail the second time you ran in the absence of `-p`.

Reply:
Ah shoot. Will fix.